lib/bike_brigade_web/live/program_live/form_component.html.heex

```diff
@@ -81,45 +81,36 @@
       <%= label f, :default_item_id, class: "block text-sm font-medium leading-5 text-gray-700" do %>
         Items
       <% end %>
-      <table class="min-w-full table-fixed">
-        <thead>
-          <tr>
-            <th class="w-1/3 px-6 py-3 text-xs font-medium leading-4 tracking-wider text-left text-gray-500 uppercase border-b border-gray-200 bg-gray-50">
-              Name
-            </th>
-            <th class="w-full px-6 py-3 text-xs font-medium leading-4 tracking-wider text-left text-gray-500 uppercase border-b border-gray-200 bg-gray-50">
-              Description
-            </th>
-              <th class="w-1/3 px-6 py-3 text-xs font-medium leading-4 tracking-wider text-left text-gray-500 uppercase border-b border-gray-200 bg-gray-50">
-              Category
-            </th>
-            <th class="flex justify-end w-32 px-3 py-3 border-b border-gray-200 bg-gray-50">
-              <C.button size={:small} patch_to={Routes.program_index_path(@socket, :new_item, @program.id)}>
-                New Item
-              </C.button>
-            </th>
-          </tr>
-        </thead>
-        <tbody class="bg-white">
-          <%= for item <- @program.items do %>
-            <tr id={"item-#{item.id}"}>
-              <td class="px-6 py-4 text-sm leading-5 text-gray-500 border-b border-gray-200">
-                <%= item.name %>
-              </td>
-              <td class="px-6 py-4 text-sm leading-5 text-gray-500 border-b border-gray-200">
-                <%= item.description %>
-              </td>
-              <td class="px-6 py-4 text-sm leading-5 text-gray-500 border-b border-gray-200">
-                <%= item.category %>
-              </td>
-              <td class="px-6 py-4 text-sm font-medium leading-5 text-right border-b border-gray-200 whitespace-nowrap">
-              <%= live_patch "Edit", to: Routes.program_index_path(@socket, :edit_item, @program.id, item.id), class: "text-indigo-600 hover:text-indigo-900" %>
-              <%#= link "Delete", to: "#", phx_click: "delete", phx_value_id: item.id, data: [confirm: "Are you sure?"], class: "pl-2 text-indigo-600 hover:text-indigo-900"  %>
-            </td>
-            </tr>
-          <% end %>
-        </tbody>
-      </table>
+      <UI.table class="table-fixed" rows={@program.items }>
+        <:th class="w-1/3">
+          Name
+        </:th>
+        <:th class="w-full">
+          Description
+        </:th>
+        <:th class="w-1/3">
+          Category
+        </:th>
+        <:th class="flex justify-end w-32" padding="px-3 py-3" uppercase={false}>
```

Comment:
curious why `uppercase` and `padding` are variables vs the other extra "custom" classes that seem to just get appended?

Reply:
The implementation might make it clearer: https://github.com/bikebrigade/dispatch/pull/40/files#diff-7792a950d4b26c637bba8528adf397d20b1eab976a175e9922b382fe1328df7eR209

tailwind classes just get appended as you said, so if we want to change padding and put it in the `class` field we'd end up with extra `p-X`'s. Same with uppercase, I want it default for all headers except when you say no.

I don't think this is the clearest way to do it but I couldn't find a better one :(